06-building-a-program-start-to-finish/writing-tests/output/invertedIndexTests.js

```diff
@@ -43,16 +43,15 @@ describe('Tests the invertedIndex.js class: ', () => {
     });
   });
 
-  describe('Checks if value returned by getIndexedWords()', () => {
-    const returnedObject = invertedIndexTrue.getIndexedWords();
+  describe('Checks getIndexedWords(), contentToDisplay(), displayInTableFormat() and searchIndexedWords()',
```

Comment:
Not sure you clearly understood what was been referred to...

You had two different describe function block had specifications for **getIndexedWords()** 

Could you have had just one describe function for  **getIndexedWords()** instead of two. What you have done now is not what that comment was targeted at.